Apps/FCM/ios/FCMSampleApp/AppDelegate.mm

```diff
@@ -1,4 +1,8 @@
 #import "AppDelegate.h"
+#import <FCMSampleApp-Swift.h>
+#import <React/RCTLinkingManager.h>
+#import <React/RCTBundleURLProvider.h>
+#import <FirebaseCore.h>
```

Comment:
These imports were moved here from `AppDelegate.h` file. 

These imports are not mentioned in our SDK docs code samples for a valid `AppDelegate.h` file. I think it's best for our sample apps to align with our docs so I moved imports to this file.